tests/unit/test_file_output.py

```
@@ -0,0 +1,132 @@
+"""Unit tests for File output type."""
+import time
+from secator.output_types import File
+
+
+def test_file_creation():
+	"""Test basic File output type creation."""
+	f = File(
+		path='/tmp/test.txt',
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/test.txt"

(S108)